src/migration/1700735663192-AdvancedSettings.ts

```diff
@@ -0,0 +1,14 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AdvancedSettings1700735663192 implements MigrationInterface {
+    name = 'AdvancedSettings1700735663192'
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        await queryRunner.query(`ALTER TABLE "advanced_settings" ADD "group" text NOT NULL DEFAULT 'advanced'`);
+        await queryRunner.query(`ALTER TABLE "advanced_settings" ALTER COLUMN "group" DROP DEFAULT`);
```

Comment:
Why?

Reply:
For me to be able to set the new column as not null, I had to add default value as adding it in a new query fails.

Alternatively set it as nullable, set values, then set column as not null.

Could also leave the default value?

Reply:
Let's leave the default value, makes sense :)

Reply:
```suggestion
```